src/thunderbots_msgs/msg/RobotStatus.msg

```diff
@@ -0,0 +1,82 @@
+
+# The rough maximum full-scale deflection of the laser sensor.
+float64 break_beam_scale
+
+# The maximum possible kick speed, in m/s.
+float64 kick_speed_max
+
+
+# The kick resolution for HScale.
+float64 kick_speed_resolution
+
+
+# The maximum possible chip distance, in m.
+float64 chip_distance_max
+
+
+# The chip resolution for HScale.
+float64 chip_distance_resolution
+
+
+# The maximum power level understood by the direct_dribbler function.
+uint32 direct_dribbler_max
+
+
+# Whether or not the robot is currently responding to radio communication.
+bool alive
+
+
+# Whether the robot is in direct mode.
+bool direct_control
+
+
+# Whether or not the ball is interrupting the robot’s laser beam.
+bool ball_in_beam
+
+
+# Whether or not the robot’s capacitor is charged enough to kick the ball.
+bool capacitor_charged
+
+
+# The voltage on the robot’s battery, in volts.
+float64 battery_voltage
+
+
+# The voltage on the robot’s kicking capacitor, in volts.
+float64 capacitor_voltage
+
+
+# The reading of the robot’s laser sensor.
+float64 break_beam_reading
+
+
+# The temperature of the robot’s dribbler motor, in degrees Celsius.
+float64 dribbler_temperature
+
+
+# The speed of the robot’s dribbler motor, in revolutions per minute.
+int32 dribbler_speed
+
+
+# The temperature of the robot’s mainboard, in degrees Celsius.
+float64 board_temperature
+
+
+# The link quality of the last received packet, from 0 to 1.
```

Comment:
Is 1 good? or is 0 good? (I'm guessing 1, but might as well be super clear here)

Reply:
left a comment